src/components/Todo/TodoItem.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+interface TodoProps {
+  todo: Todo,
+}
+
+export const TodoItem: React.FC<TodoProps> = ({ todo }) => {
+  const {
+    completed,
+    title,
+  } = todo;
+
+  return (
+    <div
+      className={classNames('todo', {
+        completed,
+      })}
+    >
+      <label className="todo__status-label">
+        <input
+          aria-label="Enter to do"
+          type="checkbox"
+          className="todo__status"
+          checked={completed}
+        />
+      </label>
+
+      <span className="todo__title">{title}</span>
+      <button type="button" className="todo__remove">×</button>
+
+      <div className="modal overlay">
+        <div className="modal-background has-background-white-ter" />
+        <div className="loader" />
+      </div>
```

Comment:
Should we show this modal all the time ?

Reply:
done